site/source/docs/porting/simd.rst

```diff
@@ -16,30 +19,718 @@ Separate documentation for the intrinsics header is a work in progress, but its
-
-There are two different ways to generate code using the SIMD.js polyfill:
-
-- The easiest way is to utilize LLVM autovectorization support. This does not require any code changes and it is the most straightforward option to try for codebases that have not explicitly been written to target SIMD. To enable autovectorization, pass the linker flag -s SIMD=1 both when compiling and linking output files.
-
-- Emscripten supports the GCC/Clang compiler specific `SIMD Vector Extensions <https://gcc.gnu.org/onlinedocs/gcc/Vector-Extensions.html>`_. These constructs do not require any changes to the command line build flags, but any code that utilizes the vector built-ins will always unconditionally emit SIMD.js vector instructions.
-
 ======================================
 Limitations and behavioral differences
 ======================================
 
-When porting native SIMD code, it should be noted that because of portability concerns, neither the WebAssembly SIMD proposal nor SIMD.js expose the full native instruction sets. In particular the following changes exist:
+When porting native SIMD code, it should be noted that because of portability concerns, the WebAssembly SIMD specification does not expose the full native instruction sets. In particular the following changes exist:
 
  - Emscripten does not support x86 or any other native inline SIMD assembly or building .s assembly files, so all code should be written to use SIMD intrinsic functions or compiler vector extensions.
 
- - WebAssembly SIMD and SIMD.js do not have control over managing floating point rounding modes or handling denormals.
+ - WebAssembly SIMD does not have control over managing floating point rounding modes or handling denormals.
 
- - Cache line prefetch instructions are not available, and calls to these functions will compile, but be treated as no-ops.
+ - Cache line prefetch instructions are not available, and calls to these functions will compile, but are treated as no-ops.
 
  - Asymmetric memory fence operations are not available, but will be implemented as fully synchronous memory fences when SharedArrayBuffer is enabled (-s USE_PTHREADS=1) or as no-ops when multithreading is not enabled (default, -s USE_PTHREADS=0).
 
 SIMD-related bug reports are tracked in the `Emscripten bug tracker with the label SIMD <https://github.com/emscripten-core/emscripten/issues?q=is%3Aopen+is%3Aissue+label%3ASIMD>`_.
+
+=====================================================
+Compiling SIMD code targeting x86 SSE instruction set
+=====================================================
+
+Emscripten supports compiling existing x86 SSE utilizing codebases by passing the `-msse` directive to the compiler, and including the header `<xmmintrin.h>`.
```

Comment:
This would read better as "compiling existing codebases utilizing x86 SSE"

Reply:
or "compiling existing codebases that use x86 SSE"